.github/workflows/ci.yaml

```diff
@@ -22,21 +24,25 @@ permissions:
   packages: write # to upload container
 
 jobs:
-
   metadata:
     runs-on: ubuntu-22.04
     outputs:
-      TRIAL_BRANCH: ${{ steps.meta.outputs.TRIAL_BRANCH }}
+      BUILD_DATE: ${{ steps.date.outputs.BUILD_DATE }}
+      PUBLISH: ${{ steps.if-publish.outputs.PUBLISH }}
     steps:
-      - name: Set optional trial branch
-        id: meta
+      - name: Set build date
+        id: date
         shell: bash -x -e {0}
         run: |
-          if [[ -z "${{ inputs. TRIAL_BRANCH}}" ]]; then
-            echo "TRIAL_BRANCH=${{ github.sha }}"
-          else
-            echo "TRIAL_BRANCH=${TRIAL_BRANCH}"
-          fi | tee $GITHUB_OUTPUT
+          BUILD_DATE=$(TZ='US/Los_Angeles' date '+%Y-%m-%d')
+          echo "BUILD_DATE=${BUILD_DATE}" >> $GITHUB_OUTPUT
+
+      - name: Determine whether results will be 'published'
+        id: if-publish
+        shell: bash -x -e {0}
+        run: |
+          echo "PUBLISH=${{ github.event_name == 'schedule' || (github.event_name == 'workflow_dispatch' && inputs.PUBLISH) }}" >> $GITHUB_OUTPUT
```

Comment:
You are missing `on.workflow_dispatch.inputs.PUBLISH`

Reply:
Is this already resolved?

Reply:
It doesn't look like it to me. The only workflow_dispatch input is TRIAL_BRANCH, but I don't see PUBLISH

Reply:
Added PUBLISH to inputs in `workflow_dispatch`